src/templates/project-details.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Layout from '../components/Layout';
+import Image from '../components/LazyImage'
+import CloudinaryImage from '../components/CloudinaryImage';
+import s from './styles.module.scss';
+
+
+const GithubButton = ({ text, className, href, ...rest }) => (
```

Comment:
We already use this button elsewhere. Please consider creating a separate component (e.g `components/GitHubButton`) so you can reuse in both places